composer.json

```diff
@@ -18,7 +18,7 @@
     ],
     "require": {
         "php": ">=5.3.2",
-        "composer/composer": "1.*@dev",
+        "composer/composer": "1.0.x-dev",
```

Comment:
It's not preferable, but all the other combinations keep getting composer/composer stuck on alpha9. The other option is dev-master.

This is also partially so @simensen can see what I did to get it to work. We might dump this PR for something else, but it's a start.